diff --git a/node_modules/create-tinyss/bin/cli.mjs b/node_modules/create-tinyss/bin/cli.mjs
index 4adc43e..e07f6c0 100755
--- a/node_modules/create-tinyss/bin/cli.mjs
+++ b/node_modules/create-tinyss/bin/cli.mjs
@@ -2140,6 +2140,7 @@ async function buildToMap(options) {
   }
   await runner.run("beforeRender", ctx);
   const outputMap = await renderToMap(tree, { ...config, _pages: ctx.pages });
+  ctx.pages = relativizePages(ctx.pages, findRootDir(tree));
   await runner.run("afterRender", ctx);
   return outputMap;
 }
